tests/console/check_package_version.pm

```diff
@@ -51,7 +51,7 @@ sub cmp_version {
 sub cmp_packages {
     my ($pcks, $pckv, $jsc) = @_;
     record_info($pcks, "$pcks version check after migration");
-    my $output = script_output("zypper se -s $pcks | grep -w $pcks | head -1 | awk -F '|' '{print \$4}'", proceed_on_failure => 1, 100);
+    my $output = script_output("zypper se -s $pcks | grep -w '| $pcks' | head -1 | awk -F '|' '{print \$4}'", proceed_on_failure => 1, 100);
```

Comment:
if you want the exact match, better pass `-x` to `zypper search`

Reply:
Yes, It's better. Thanks